pkg/operator/ceph/cluster/mgr/mgr_test.go

```diff
@@ -176,130 +170,6 @@ func validateServices(t *testing.T, c *Cluster) {
 	}
 }
 
-func TestUpdateServiceSelectors(t *testing.T) {
```

Comment:
Similar to keeping the `updateServiceSelectors()` method and updating its implementation, we'll want to keep this unit test and similarly update it.

Reply:
Just put some UT to test the new behaviour.